spec/Section 5 -- Validation.md

```diff
@@ -1494,44 +1520,6 @@ fragment HouseTrainedFragment {
 ```
 
 
-### Variable Default Value Is Allowed
```

Comment:
This rule is completely removed since the queries it prohibited we explicitly wish to support.